examples/hello_flutter/BUILD.gn

```diff
@@ -4,7 +4,7 @@
 
 import("//flutter/build/flutter_app.gni")
 
-flutter_app("hello_flutter") {
+flutter_jit_app("hello_flutter") {
```

Comment:
The intent is to ensure there's at least one app built both ways. There's a flutter_aot_app just below here.